vdr/vdr.go

```diff
@@ -81,17 +81,13 @@ func (r *VDR) Config() interface{} {
 	return &r.config
 }
 
-// Start initiates the routines for auto-updating the data
-func (r *VDR) Start() error {
+// Configure configures the VDR engine.
+func (r *VDR) Configure(_ core.ServerConfig) error {
```

Comment:
I underdstand the fix, but I don't like that the network is replaying its events before all subscribers are ready. We should rewrite the subscriber meganism so that each subscription triggers its own replay from a given point.

Reply:
I swear I though that was an existing issue to change it to something like NATS streaming, but can't find it.

For now: since it was decided to make the DID Store in-memory, we don't have much choice than just to replay the complete DAG. Also, we don't have other subscribers, so there's no-one requiring replays from a given point (although I see the feature for in the future :-p)